sdk/maps/azure-maps-render/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0-beta.1 (2022-09-06)
```

Comment:
Is this the planned date of the first preview release?

Reply:
Yes this is the planned date of the first preview release for September!